core/templates/dev/head/pages/exploration_editor/editor_tab/StateInteraction.js

```diff
@@ -397,3 +397,31 @@ oppia.directive('testInteractionPanel', [function() {
     ]
   };
 }]);
+
+oppia.directive('heightWatcher', [function() {
+  return {
+    restrict: 'A',
+    link:function(scope, element, attrs) {
+    scope.$watch(
+      function() {
+        return element[0].offsetHeight;
+      },
+      function(newVal,oldVal) {
+        if (newVal!==oldVal) {
```

Comment:
I would be surprised if this passed lint checks. Please submit code via the command line.